traffic_ops/testing/api/test/wrapper.go

```diff
@@ -0,0 +1,122 @@
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package test
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+
+	tc "github.com/apache/incubator-trafficcontrol/lib/go-tc"
```

Comment:
This isn't hurting anything, but it doesn't strictly need the alias, Go will automatically import `go-tc` as `tc`.

Reply:
fixed